packages/css-framework/src/components/_transfer.scss

```diff
@@ -0,0 +1,59 @@
+.rn-transfer {
+  .column-left,
+  .column-right {
+    width: 40%;
+    float: left;
+    border: 1px solid #e0e8ed;
+    color: #012159;
+    label {
+      span {
+        display: block;
+        padding: 10px;
+        margin-bottom: 1px;
+        &:hover {
+          background: #e9f8ff;
+        }
+      }
+      input {
+        display: none;
+        &:checked + span {
+          background: #e9f8ff;
+        }
+      }
+    }
+  }
+  .transfer-buttons {
+    width: 10%;
+    float: left;
+    margin-top: 90px;
+
+    .left, .right {
```

Comment:
Are there any individual styles for these classes? If not, we can refactor them into a generic `button` class.